src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2274,8 +2274,18 @@ because xbuild doesn't support framework reference assemblies.
   </PrepareAbiItems>
 </Target>
 
+<PropertyGroup>
+  <_GenerateJavaStubsDependsOnTargets>
+    _SetLatestTargetFrameworkVersion;
+    _PrepareNativeAssemblySources;
+    _LinkAssemblies;
+    _PrepareAssemblies;
+    $(_AfterPrepareAssemblies);
```

Comment:
I see this property is actually used in the Debugging.props and overwrites the value

Reply:
Yup we need to alter the `Xamarin.Android.Common.Debugging.props` from

```
<_AfterPrepareAssemblies>
	_WriteResolvedAssemblies
</_AfterPrepareAssemblies>
```

to 

```
<_AfterPrepareAssemblies>
	$(_AfterPrepareAssemblies);
	_WriteResolvedAssemblies
</_AfterPrepareAssemblies>
```

Reply:
Should be fixed in https://github.com/xamarin/monodroid/pull/990

Reply:
The fix above was merged